sandbox/lua/modules/line_protocol.lua

```diff
@@ -0,0 +1,246 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Sounds great. And thanks so much for staying with us through all of the feedback. :)